src/helpers/datetime.js

```diff
@@ -56,43 +57,6 @@ export const defaultDateTimeEnd = moment
 	.add(2, 'hours')
 	.format(dateTimeDatabaseFormat);
 
-/**
- * Predefined duration options for event scheduling.
- *
- * This array contains a list of duration options in hours that can be selected
- * for an event. Each option includes a label for display and a corresponding
- * value representing the duration in hours. The last option allows the user
- * to set a custom end time by selecting `false`.
- *
- * @since 1.0.0
- *
- * @type {Array<Object>} durationOptions
- * @property {string}         label - The human-readable label for the duration option.
- * @property {number|boolean} value - The value representing the duration in hours, or `false` if a custom end time is to be set.
- */
-export const durationOptions = [
```

Comment:
let's keep the name `durationOptions`, but convert this to a function. Then apply your filter to the function before returning.